internal/display/quickstarts.go

```diff
@@ -0,0 +1,62 @@
+package display
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/auth0/auth0-cli/internal/auth0"
+)
+
+type quickstartView struct {
+	Stack   string
+	AppType string
+	URL     string
+}
+
+func (v *quickstartView) AsTableHeader() []string {
+	return []string{"Quickstart", "Type", "URL"}
+}
+
+func (v *quickstartView) AsTableRow() []string {
+	return []string{v.Stack, applyColor(v.AppType), v.URL}
+}
+
+func (r *Renderer) QuickstartList(qs map[string][]auth0.Quickstart) {
+	r.Heading()
+
+	var results []View
+	keys := make([]string, 0, len(qs))
+
+	for key := range qs {
+		keys = append(keys, key)
+	}
+
+	sort.Strings(keys)
+
+	for _, key := range keys {
+		for _, item := range qs[key] {
+			results = append(results, &quickstartView{
+					Stack: item.Name,
+					AppType: applyColor(qsAppTypeFor(key)),
+					URL: fmt.Sprintf("https://auth0.com/docs/quickstart/%s/%s", key, item.Path),
```

Comment:
nit: it can be a const at the top to be (slightly) easier to change.

Reply:
Good catch, changed in https://github.com/auth0/auth0-cli/pull/254/commits/1a95a65d85fd00ebc86ce64d201949571d954585